e2e/wdio/platform/tests/button.e2e-spec.ts

```diff
@@ -0,0 +1,74 @@
+import { ButtonPo } from '../pages/button.po';
+import {
+    getAttributeByName,
+    getElementArrayLength,
+    getElementTitle,
+    isElementClickable,
+    scrollIntoView,
+} from '../../driver/wdio';
+
+describe('Button test suite:', function() {
+    const buttonPage = new ButtonPo();
+    const {
+        typeButtons, sizeButtons, iconButtons, stateButton, disableStateButtons, truncatedButton  } = buttonPage;
+
+    beforeAll(() => {
+        buttonPage.open();
+    }, 1);
+
+    it('Verify all buttons are clickable', function() {
```

Comment:
remove.
`it` is for a specific test case, cannot contain other `it` inside it. breaks test.